### PR TITLE
[typescript-axios]: handle explode query

### PR DESCRIPTION
**Pull request by tienipia**
### PR checklist
 
- [x] Read the [contribution guidelines](https://github.com/openapitools/openapi-generator/blob/master/CONTRIBUTING.md).
- [x] Pull Request title clearly describes the work in the pull request and Pull Request description provides details about how to validate the work. Missing information here may result in delayed response from the community.
- [x] Run the following to [build the project](https://github.com/OpenAPITools/openapi-generator#14---build-projects) and update samples:
  ```
  ./mvnw clean package 
  ./bin/generate-samples.sh
  ./bin/utils/export_docs_generators.sh
  ``` 
  Commit all changed files. 
  This is important, as CI jobs will verify _all_ generator outputs of your HEAD commit as it would merge with master. 
  These must match the expectations made by your contribution. 
  You may regenerate an individual generator by passing the relevant config(s) as an argument to the script, for example `./bin/generate-samples.sh bin/configs/java*`. 
  For Windows users, please run the script in [Git BASH](https://gitforwindows.org/).
- [x] In case you are adding a new generator, run the following additional script : 
  ```
  ./bin/utils/ensure-up-to-date
  ``` 
  Commit all changed files.
- [x] File the PR against the [correct branch](https://github.com/OpenAPITools/openapi-generator/wiki/Git-Branches): `master` (7.0.1 - patch release), `7.1.x` (minor release - breaking changes with fallbacks), `8.0.x` (breaking changes without fallbacks)
- [x] If your PR is targeting a particular programming language, @mention the [technical committee](https://github.com/openapitools/openapi-generator/#62---openapi-generator-technical-committee) members, so they are more likely to review the pull request.

Since the original issue(#15633) and PR(#15641)  don't seem to be progressing, I'm trying to work again to resolve the issue.

### CODE REVIEW

**Comment by tienipia** [comment]
@macjohnny
I applied the review of #15641 pull request.
Please check this issue to fix issue.

Thank you.

File: modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache [hunk 1]
```diff
@@ -127,7 +127,20 @@ export const {{classname}}AxiosParamCreator = function (configuration?: Configur
                     {{paramName}};
                 {{/isDate}}
                 {{^isDate}}
+                {{#isExplode}}
+                {{#isPrimitiveType}}
                 localVarQueryParameter['{{baseName}}'] = {{paramName}};
+                {{/isPrimitiveType}}
+                {{^isPrimitiveType}}
+                let param: keyof typeof {{baseName}};
```
Comment (macjohnny) [inline modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache:135]:
```suggestion
                let param: keyof typeof {{baseName}};
```

please also re-generate the samples

**Review by macjohnny — CHANGES_REQUESTED** [review]
thanks for your contribution!

File: modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache (hunk 1 above)
Comment (tienipia) [reply modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache:135]: No any changes after re-generating samples.
What I've done is, (under current commit)
```bash
mvn clean package
./bin/generate-samples.sh ./bin/configs/typescript-axios*
# No any changes after execute generate samples command lines.
```
Did I do something wrong?

Comment (macjohnny) [inline modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache:135]:
```suggestion
```

Comment (macjohnny) [reply modules/openapi-generator/src/main/resources/typescript-axios/apiInner.mustache]: ok then probably there were no configs for this use case, the CI is green, so all good.

**Pull request merged**